.gitlab-ci.yml

```diff
@@ -400,6 +403,11 @@ examples-contract-build:
     - cargo contract -V
     - for example in integration-tests/*/; do
         if [ "$example" = "integration-tests/lang-err-integration-tests/" ]; then continue; fi;
+        if [ "$example" = "integration-tests/conditional-compilation/" ]; then
+        pushd $example &&
+        cargo +stable contract build --features "foo, bar" &&
```

Comment:
Maybe you want to compile it with different combinations of features? Like only with `foo`, only with `bar` and both?

Reply:
Done